app/views/control_panel/stories/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_for([@story, @story], url: control_panel_board_story_path) do |f| %>
```

Comment:
it looks weird :D 
IMO it should be `[:contol_panel, @board, @story]` and you can remove url ;)
Read source code of form_for you find out what is going on there. You should use param url only in situation when you have custom behaviour. That is not a special case ;)